modules/bustools/sort/meta.yml

```diff
@@ -0,0 +1,48 @@
+name: bustools_sort
+## TODO nf-core: Add a description of the module and list keywords
```

Comment:
Be great if you can sort this file out @tkhalilullah @Emiller88 and then I think it is pretty much ready to go. Can update it properly later if required.